client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -25,7 +25,7 @@ import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
 import { bumpStat, composeAnalytics, recordTracksEvent } from 'calypso/state/analytics/actions';
 import ActionBox from './action-box';
 import isHomeQuickLinksExpanded from 'calypso/state/selectors/is-home-quick-links-expanded';
```

Comment:
Now when we toggle the Quick Links by direct calls to `savePreference`, this selector could be also inlined as a direct `getPreference` selector call.